sway-core/src/semantic_analysis/ast_node/code_block.rs

```diff
@@ -47,33 +47,40 @@ impl TypedCodeBlock {
                 _ => None,
             })
             .flatten();
+        let span = implicit_return_span.unwrap_or_else(|| code_block.whole_block_span.clone());
 
-        // find the implicit return, if any, and use it as the code block's return type.
-        // The fact that there is at most one implicit return is an invariant held by the parser.
-        let return_type = evaluated_contents.iter().find_map(|x| match x {
-            TypedAstNode {
-                content:
-                    TypedAstNodeContent::ImplicitReturnExpression(TypedExpression {
-                        ref return_type,
-                        ..
-                    }),
-                ..
-            } if !x.deterministically_aborts() => Some(*return_type),
-            _ => None,
-        });
+        let block_type =
+            {
+                // find the implicit return, if any, and use it as the code block's return type.
```

Comment:
Because I write code the way I think looks sensible then I run rustfmt before pushing in order to "fix" it and sometimes it outputs stuff like that. I've fixed it now.